src/dsc/Microsoft.PowerToys.Configure/examples/allSettings.dsc.yaml

```diff
@@ -0,0 +1,270 @@
+          ImageresizerTiffCompressOption: 187
+          ImageresizerFileName: "<string>"
+          ImageresizerKeepDateModified: "<string>"
+          ImageresizerFallbackEncoder: "<string>"
+
+        MeasureTool:
+          ActivationShortcut: "<string>"
+          ContinuousCapture: true
+          DrawFeetOnCross: false
+          PerColorChannelEdgeDetection: false
+          PixelTolerance: 175
+          MeasureCrossColor: "<string>"
+          DefaultMeasureStyle: 53
+
+        MouseHighlighter:
+          ActivationShortcut: "<string>"
+          LeftButtonClickColor: "<string>"
+          RightButtonClickColor: "<string>"
+          AlwaysColor: "<string>"
+          HighlightRadius: 132
+          HighlightFadeDelayMs: 124
+          HighlightFadeDurationMs: 64
+          AutoActivate: "<string>"
+
+        MouseJump:
+          ActivationShortcut: "<string>"
+          ThumbnailSize: "<string>"
+
+        MousePointerCrosshairs:
+          ActivationShortcut: "<string>"
+          CrosshairsColor: "<string>"
+          CrosshairsOpacity: 197
+          CrosshairsRadius: 0
+          CrosshairsThickness: 123
+          CrosshairsBorderColor: "<string>"
+          CrosshairsBorderSize: 114
+          CrosshairsAutoHide: "<string>"
+          CrosshairsIsFixedLengthEnabled: "<string>"
+          CrosshairsFixedLength: 124
+          AutoActivate: "<string>"
+
+        MouseWithoutBorders:
+          ShowOriginalUI: true
+          WrapMouse: false
+          ShareClipboard: false
+          TransferFile: false
+          HideMouseAtScreenEdge: true
+          DrawMouseCursor: false
+          ValidateRemoteMachineIP: false
+          SameSubnetOnly: true
+          BlockScreenSaverOnOtherMachines: false
+          MoveMouseRelatively: true
+          BlockMouseAtScreenCorners: true
+          ShowClipboardAndNetworkStatusMessages: true
+          EasyMouse: 113
+          HotKeySwitchMachine: 127
+          ToggleEasyMouseShortcut: "<string>"
+          LockMachineShortcut: "<string>"
+          ReconnectShortcut: "<string>"
+          Switch2AllPCShortcut: "<string>"
+          DrawMouseEx: false
+          Name2IP: "<string>"
+
+        PastePlain:
+          ActivationShortcut: "<string>"
+
+        Peek:
+          ActivationShortcut: "<string>"
+          AlwaysRunNotElevated: "<string>"
+          CloseAfterLosingFocus: "<string>"
+
+        PowerAccent:
+          ActivationKey: Space
+          DoNotActivateOnGameMode: true
+          ToolbarPosition: "<string>"
+          InputTime: 17
+          SelectedLang: "<string>"
+          ExcludedApps: "<string>"
+          ShowUnicodeDescription: true
+          SortByUsageFrequency: false
+          StartSelectionFromTheLeft: false
+
+        PowerLauncher:
+          OpenPowerLauncher: "<string>"
+          IgnoreHotkeysInFullscreen: false
+          ClearInputOnLaunch: false
+          TabSelectsContextButtons: false
+          Theme: HighContrastBlack
```

Comment:
Wonder about this value. Is it matching to a selectable value in the settings ui?